engine/src/cmd/planet_generic.cpp

```diff
@@ -222,8 +222,13 @@ Planet::Planet() :
     terraintrans = NULL;
     atmospheric  = false;
     //Force shields to 0
-    memset( &(this->shield), 0, sizeof (Unit::shield) );
-    this->shield.number = 2;
+//    memset( &(this->shield), 0, sizeof (Unit::shield) ); //Let's not use memset on non-trivial stuff
```

Comment:
if we know the structure is okay to do so, then it's fine to do.

However, for non-trivial stuff it'd be better to make it a formal `class` (as opposed to a `struct`) and add a method that can be called to do this; or use a constructor to do it (for initialization).

Reply:
Thanks for the review. I'm sure to return to this code in the future, but for now it just makes the compiler happy.